### DISCUSSION TITLE
Decarbonising End Use Spec Review - standards group wg feedback

### DISCUSSION THREAD

**Discussion by mrchrisadams** [Green-Software-Foundation/software-standards-wg#101] [Ideas]
Hey folks, 

_(I'm tagging @atg-abhishek, who I still think is the chair of this working group, please let me know if there is another person I should tag in future)_

I do apologise for sending this notification so late.

We had a meeting in the policy WG earlier this week, and one of my actions was to notify the standards wg, about our feedback sent to the partnering organisation, the Carbon Trust about a spec they were developing for decarbonising end user use-phase emissions.

You can see the minutes below here:

https://github.com/Green-Software-Foundation/policy-wg/issues/66

You can see the actual spec and the rest In this Google drive directory (it's only visible to GSF staff, as we couldn't publish the draft spec yet):

https://drive.google.com/drive/folders/11nyVsVhcQMOE8RR5nsjS4XcE8jXwMTEm

In the folder we have:

1. the covering letter from the carbon trust
2. the draft spec (it's 57 pages long!)
3. my annotated version of the spec with highlights for others to see (for convenience(
4. the form sent to us for feedback
5. the responses i have collated

I also have a call lateer this month with the Carbon Trust to discuss any further feedback with the Carbon Trust, so if there's a thing you'd like to raise with them, there's still time to let me know.

We've been requested not to share too much publiclty yet, so I'd appreciate comments either in the spreadsheet or on my email address chris@thegreenwebfoundation.org.

**When to give the feedback**

Ideally as soon as is reasonable. I've told the Carbon Trust I'm collating further feedback after gathering comments from policy wg members, and the initial date feedback was requested was Feb 9th (yes, thay's today, sorry), there is a two week period at their end to process feedback, so it's possible that any updates shared with my I can forward along. I apologise again for the late notice.

However, I also have a 1hr call booked on Feb 23rd with them - I'm happy to take any further comments along to that call if there is something we missed that would make sense to raise.

**Comment by seanmcilroy29** [comment]
adding @Henry-WattTime @navveenb

**Comment by atg-abhishek** [comment]
Hi Chris,

Thanks for sharing this - I can take a look through the files in the folder that you shared tomorrow and share any feedback that arises from it. I'm still away from work, so there might be a more nuanced perspective given the work that's happened at the SWG in the interim that Henry and Navveen can chime in with as well.

Thanks!

Abhishek Gupta ( @atg_abhishek ( https://twitter.com/atg_abhishek ) )
Founder and Principal Researcher, Montreal AI Ethics Institute ( https://montrealethics.ai/ )
Director, Responsible AI, Boston Consulting Group (BCG) ( https://www.bcg.com/about/people/experts/abhishek-gupta )
Fellow, Augmented Collective Intelligence, BCG Henderson Institute ( https://bcghendersoninstitute.com/contributors/abhishek-gupta/ )
Author, State of AI Ethics Report ( https://montrealethics.ai/state ) and AI Ethics Brief ( https://brief.montrealethics.ai/ )
Chair, Standards Working Group, Green Software Foundation ( https://greensoftware.foundation/ )
Learn more about my work here ( https://atg-abhishek.github.io/ ). You can now support my work by buying me a coffee ( https://buymeacoff.ee/abhishekgupta ) !
